diff -up doxygen-1.9.4/src/CMakeLists.txt.2~ doxygen-1.9.4/src/CMakeLists.txt
--- doxygen-1.9.4/src/CMakeLists.txt.2~	2022-05-08 16:33:14.035356647 +0200
+++ doxygen-1.9.4/src/CMakeLists.txt	2022-05-08 17:12:22.952670916 +0200
@@ -343,7 +343,7 @@ if (use_libclang)
         set(CLANG_LIBS libclang clangTooling)
         add_definitions(-DCINDEX_NO_EXPORTS)
     else() # dynamically linked version of clang
-        llvm_config(doxymain USE_SHARED support)
+        #llvm_config(doxymain USE_SHARED support)
         set(CLANG_LIBS libclang clang-cpp)
     endif()
     target_compile_definitions(doxygen PRIVATE ${LLVM_DEFINITIONS})
