diff -up doxygen-1.8.14/src/clangparser.cpp.omv~ doxygen-1.8.14/src/clangparser.cpp
--- doxygen-1.8.14/src/clangparser.cpp.omv~	2018-04-28 19:49:11.196968640 +0200
+++ doxygen-1.8.14/src/clangparser.cpp	2018-04-28 19:49:29.989539464 +0200
@@ -5,6 +5,7 @@
 #if USE_LIBCLANG
 #include <clang-c/Index.h>
 #include "clang/Tooling/Tooling.h"
+#include <clang/Tooling/CompilationDatabase.h>
 #include <qfileinfo.h>
 #include <stdlib.h>
 #include "message.h"
diff -up doxygen-1.8.14/src/CMakeLists.txt.0003~ doxygen-1.8.14/src/CMakeLists.txt
--- doxygen-1.8.14/src/CMakeLists.txt.0003~	2017-10-31 20:36:00.000000000 +0100
+++ doxygen-1.8.14/src/CMakeLists.txt	2018-04-28 21:05:53.114433953 +0200
@@ -260,7 +260,7 @@ if (use_libclang)
     add_definitions(${LLVM_DEFINITIONS})
     llvm_map_components_to_libnames(llvm_libs support core option)
     target_compile_definitions(doxygen PRIVATE ${LLVM_DEFINITIONS})
-    set(CLANG_LIBS libclang clangTooling ${llvm_libs})
+    set(CLANG_LIBS libclang clangTooling clangBasic ${llvm_libs})
 endif()
 
 target_link_libraries(doxygen
