diff --git a/Makefile.in b/Makefile.in
index 1cd4f16..a281342 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -53,7 +53,7 @@ distclean: clean
 
 DATE=$(shell date "+%B %Y")
 
-MAN1DIR = man/man1
+MAN1DIR = share/man/man1
 
 install: doxywizard_install doxysearch_install
 	$(INSTTOOL) -d $(DESTDIR)/$(INSTALL)/bin
diff --git a/addon/doxywizard/Makefile.in b/addon/doxywizard/Makefile.in
index 727409a..406429a 100644
--- a/addon/doxywizard/Makefile.in
+++ b/addon/doxywizard/Makefile.in
@@ -28,6 +28,8 @@ distclean: Makefile.doxywizard
 	$(MAKE) -f Makefile.doxywizard distclean 
 	$(RM) Makefile.doxywizard
 
+MAN1DIR = share/man/man1
+
 install:
 	$(INSTTOOL) -d $(INSTALL)/bin	
 	$(INSTTOOL) -m 755 ../../bin/doxywizard $(INSTALL)/bin	
